scripts/configurator/parser/cmake_row.py

```diff
@@ -32,33 +48,33 @@ def __init__(self, raw_row: str, row_number: int):
       """
       row  = raw_row.strip()
       matches: Optional[Match[str]]  = re.match(self.regex_expression, row)
-      self.type: str =  self.NOT_PARSABLE
+      self.type = row_type.NOT_PARSABLE
       self.val: str = ""
       self.name: str = ""
       self.var_type: str = ""
       self.row_number: int = row_number
       if not len(row):
-         self.type = self.EMPTY
+         self.type = row_type.EMPTY
          return
 
       if self.is_comment(row):
          if self.is_slash_comment(row):
-            self.type = self.SLASH_COMMENT
+            self.type = row_type.SLASH_COMMENT
             self.val = row[2:]
             return
 
-         self.type = self.HASHTAG_COMMENT
+         self.type = row_type.HASHTAG_COMMENT
          self.val = row[1:]
          return
 
       elif matches: # this guarantees that the regex has exactly 3 matches
          groups = matches.groups()
          self.name = groups[0]
-         self.type = self.VARIABLE
+         self.type = row_type.VARIABLE
          self.val = groups[2]
-         self.var_type = groups[1]
-         if self.var_type == "BOOL":
-            self.val = self.BOOL_MAPPINGS[self.val]
+         self.var_type = self.STR_TO_ENUM[groups[1]]
```

Comment:
And here you're treating it as an object. Did you run mypy? This shouldn't pass.

Reply:
You're right. I will fix this

Reply:
> Sure, i will squash, but this is not the final shape of the code yet. I will ask for review again when it's actually ready to be reviewed.